scripts/release/mule/package/docker/package.sh

```diff
@@ -6,12 +6,16 @@ echo
 date "+build_release begin PACKAGE DOCKER stage %Y%m%d_%H%M%S"
 echo
 
-ARCH_TYPE=$(./scripts/archtype.sh)
-OS_TYPE=$(./scripts/ostype.sh)
-BRANCH=${BRANCH:-$(./scripts/compute_branch.sh "$BRANCH")}
-CHANNEL=${CHANNEL:-$(./scripts/compute_branch_channel.sh "$BRANCH")}
-PKG_ROOT_DIR="./tmp/node_pkgs/$OS_TYPE/$ARCH_TYPE"
+if  [ -z "$NETWORK" ]; then
+    echo "[$0] NETWORK=$NETWORK is missing."
```

Comment:
Same here, message will always be `NETWORK= is missing."`

Reply:
Fixed these everywhere.